src/config/limits.ts

```
@@ -1,25 +1,177 @@
+        config.limits.circuitBreaker.reserveWeightThresholdPct,
+      circuitBreakerMinReserveRatio:
+        config.limits.circuitBreaker.minReserveRatio,
+    },
+  };
+
+  for (const limit of limits) {
+    const audience = limit.audience as Audience;
+    if (result[audience]) {
+      result[audience] = {
+        ...result[audience],
+        depositDailyUsd: Number(limit.depositDailyUsd) || result[audience].depositDailyUsd,
```

ESLint

[error] Insert `⏎·········`

(prettier/prettier)